c7n/schema.py

```diff
@@ -414,12 +414,36 @@ def resource_vocabulary(cloud_name=None, qualify_name=True):
     for type_name, resource_type in resources.items():
         classes = {'actions': {}, 'filters': {}, 'resource': resource_type}
         actions = []
+        seen_actions = set() # Aliases get processed once
         for action_name, cls in resource_type.action_registry.items():
+
+            if cls in seen_actions:
+                continue
+            else:
+                seen_actions.add(cls)
+
+            if 'type' in cls.schema['properties']:
+                action_name = cls.schema['properties']['type']['enum'][0]
+            else:
+                import pdb; pdb.set_trace()
```

Comment:
stray breakpoint

Reply:
yeah.. i've got a few prints in here as well.. i added a todo list to the pr summary.

Reply:
also fwiw, both of these are on the flake8 list for multi statement lines.